### PR TITLE
ZK-3920: listbox wrong scroll-height caused by focus-a position

### PR DESCRIPTION
**Pull request by klyvechen**

### CODE REVIEW

File: zktest/src/archive/test2/config.properties [hunk 1]
```diff
@@ -2457,6 +2457,7 @@ B85-ZK-3848.zul=A,E,DefinitionLoader,land-addon
 ##zats##B85-ZK-3935.zul=A,E,EventQueue,Application,DesktopCleanup
 B85-ZK-3864.zul=A,E,Frozen,MeshWidget,position,z-index
 ##ztl##B85-ZK-3911.zul=A,E,BinderImpl,MatchMedia
+##ztl##B85-ZK-920.zul=A,E,listbox,scrollheight,focus-a
```
Comment (scribetw) [inline zktest/src/archive/test2/config.properties]: 3920

Comment (klyvechen) [reply zktest/src/archive/test2/config.properties]: ok

File: zktest/src/archive/test2/config.properties [hunk 2]
```diff
@@ -248,6 +248,8 @@ B30-1916473.zul=C,M,Vbox,Hbox,Splitter,IE,Dragdrop
 ##ztl##B30-1917875.zul=B,M,Vbox,Hbox,Splitter,VisionTest
 ##ztl##B30-1918796.zul=A,E,Grid,Groupbox
 ##ztl##B30-1919180.zul=B,E,Grid
+
+
```
Comment (scribetw) [inline zktest/src/archive/test2/config.properties]: ??

why two empty line

**Comment by DevChu** [comment]
@zkadmin rebase

**Comment by zkadmin** [comment]
I just pushed up the changes, enjoy!

**Pull request merged**